src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -691,7 +691,7 @@ private void applyReplayGain(VanillaMediaPlayer mp) {
 			adjust += 2*(mReplayGainBump-75)/10f; /* 2* -> we want +-15, not +-7.5 */
 		}
 
-		if(mReplayGainAlbumEnabled == false && mReplayGainTrackEnabled == false) {
+		if ((mReplayGainAlbumEnabled && mReplayGainTrackEnabled) == false) {
```

Comment:
This actually changes the condition. It was:

| x | y | result |
| - | - | - |
0|0|1
1|0|0
0|1|0
1|1|0

And it became:

| x | y | result |
| - | - | - |
|0|0|1|
|1|0|1|
|0|1|1|
|1|1|0|